lib/trollop.rb

```diff
@@ -747,6 +811,9 @@ def initialize(name, desc="", opts={}, &b)
     ## fill in :multi
     opts[:multi] ||= false
 
+    ## fill in :hidden
+    opts[:hidden] ||= false
```

Comment:
wish we didn't need so much code for an option that overrides this gem trying to be clever (by picking the short names)